charts/quickwit/values.yaml

```diff
@@ -326,53 +371,84 @@ bootstrap:
 
   # Extra env for boostrap jobs
   extraEnv: {}
-    # KEY: VALUE
+  # KEY: VALUE
   extraEnvFrom: []
     # - secretRef:
     #     name: quickwit-boostrap
     # - configMapRef:
-    #     name: quickwit-boostrap
+  #     name: quickwit-boostrap
+
+  # extraVolumes -- Additional Volumes to use with Vector Pods.
+  extraVolumes: []
+
+  # extraVolumeMounts -- Additional Volume to mount into Vector Containers.
+  extraVolumeMounts: []
 
   resources: {}
     # limits:
     #   cpu: 100m
     #   memory: 128Mi
     # requests:
     #   cpu: 100m
-    #   memory: 128Mi
+  #   memory: 128Mi
+
+  # initContainers -- Init Containers to be added to the pods
+  initContainers: []
 
   nodeSelector: {}
 
   tolerations: []
 
   affinity: {}
 
+jobs:
+  sources:
+    # initContainers -- Init Containers to be added to the pods
+    initContainers: []
+
+    # extraVolumes -- Additional Volumes to use with quickwit Pods.
+    extraVolumes: []
+
+    # extraVolumeMounts -- Additional Volume to mount into quickwit Containers.
+    extraVolumeMounts: []
+
+  indexes:
+    # initContainers -- Init Containers to be added to the pods
+    initContainers: []
+
+    # extraVolumes -- Additional Volumes to use with quickwit Pods.
+    extraVolumes: []
+
+    # extraVolumeMounts -- Additional Volume to mount into quickwit Containers.
+    extraVolumeMounts: []
 
 # Quickwit configuration
 # Warning: This config is writed directly into a configMap
 # to avoid passing sensitive value you can pass environment variables.
 # https://quickwit.io/docs/configuration/node-config#using-environment-variables-in-the-configuration
+configLocation: /quickwit/node.yaml
+
 config:
   version: 0.8
   listen_address: 0.0.0.0
   gossip_listen_port: 7282
   data_dir: /quickwit/qwdata
   default_index_root_uri: s3://quickwit/indexes
 
-  # postgres:
-  #   max_num_connections: 50
+    # postgres:
+    #   max_num_connections: 50
 
-  # storage:
+    # storage:
     # s3:
-      # endpoint: "http://custom-s3-endpoint"
-      # region: eu-east-1
-      # We recommend using IAM roles and permissions to access Amazon S3 resources,
-      # but you can specify a pair of access and secret keys if necessary.
-      # access_key_id: <my access key>
-      # secret_access_key: ${AWS_ACCESS_KEY_ID}
+    # endpoint: "http://custom-s3-endpoint"
+    # region: eu-east-1
+    # We recommend using IAM roles and permissions to access Amazon S3 resources,
+    # but you can specify a pair of access and secret keys if necessary.
+    # access_key_id: <my access key>
+    # secret_access_key: ${AWS_ACCESS_KEY_ID}
     # azure:
-      # account: "<my account name>"
-      # access_key: ${QW_AZURE_STORAGE_ACCESS_KEY}
+    # account: "<my account name>"
+  # access_key: ${QW_AZURE_STORAGE_ACCESS_KEY}
```

Comment:
Those comments shouldn't be changed as well

Reply:
Fixed